sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/DefaultTrigger.java

```diff
@@ -91,4 +91,9 @@ private boolean endOfWindowReached(Trigger.TriggerContext context) {
 
   @Override
   public void onFire(Trigger.TriggerContext context) throws Exception { }
+
+  @Override
+  public String toString() {
+    return "Repeatedly.forever(AfterWatermark.pastEndOfWindow)";
```

Comment:
This should be either:

`"DefaultTrigger"` (which may be what users expect to see, and lets us differentiate the two)
or it should reuse AfterWatermark.TO_STRING and such (specifically, your `toString` there had `pastEndOfWindow()`).